src/main/java/minegame159/meteorclient/systems/modules/render/WaypointsModule.java

```diff
@@ -33,16 +35,104 @@
 import minegame159.meteorclient.utils.render.color.Color;
 import minegame159.meteorclient.utils.world.Dimension;
 import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.gui.screen.DeathScreen;
+import net.minecraft.text.BaseText;
+import net.minecraft.text.LiteralText;
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.util.math.Vec3d;
 
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Iterator;
+import java.util.ListIterator;
+
+import static minegame159.meteorclient.utils.player.ChatUtils.formatCoords;
+
 public class WaypointsModule extends Module {
     private static final Color GRAY = new Color(200, 200, 200);
+    
+    private final SettingGroup sgDeathPosition = settings.createGroup("death-position");
+
+    private final Setting<Integer> dpWpAmouunt = sgDeathPosition.add(new IntSetting.Builder()
+            .name("wp-amount")
```

Comment:
Please name it something more descriptive

Reply:
The new name ("max-death-positions") should be more descriptive